[encode] 1 [01]
[encode] 1 [01]
[encode] 0 [10]
terminate called after throwing an instance of 'std::range_error'
  what():  length is not a multiple of 2
bash: line 7: 26302 Aborted                 (core dumped) ./a.out
